tests/integ/test_tf_script_mode.py

```diff
@@ -82,3 +85,18 @@ def _assert_s3_files_exist(s3_url, files):
         found = [x['Key'] for x in contents if x['Key'].endswith(f)]
         if not found:
             raise ValueError('File {} is not found under {}'.format(f, s3_url))
+
+
+def _assert_s3_files_exist_in_tar(s3_url, files):
```

Comment:
This function is not being used anywhere.

Reply:
Remoived